### PR TITLE
Removes spaces in stripHTMLtags module.

### PR DESCRIPTION
**Pull request by terw-dan**
The stripHTMLtags module replaces htmltags with spaces, but this might give double spaces when multiple tags are removed. 
Adds stripSpaces after replacing the HTMLtags so no double spaces are included in the outputString. 

Includes spec for stripHTMLTags

### CODE REVIEW

File: spec/stringProcessing/stripHTMLTagsSpec.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var stripHTMLTags = require( "../../js/stringProcessing/stripHTMLTags.js" );
+
+describe( "strips the HTMLtags from a string", function(){
+	it( "returns a string without HTMLtags", function(){
```
Comment (diedexx) [inline spec/stringProcessing/stripHTMLTagsSpec.js]: I wonder what it does with a string with multiple consecutive spaces 
(like: "this &nbsp;&nbsp; &lt;b&gt; is &nbsp;&nbsp; a &lt;/b&gt; teststring")
please add tests for this.

**Comment by diedexx** [comment]
CR 1 done :+1:

**Comment by terw-dan** [comment]
Added a test with multiple consecutive spaces by suggestion of @diedexx

**Comment by terw-dan** [comment]
the stripHTMLtags module is correct, even though it doesn't show up in the latest commit, where only the test was added.

**Pull request merged**